debian/control

```diff
@@ -73,6 +73,7 @@ Build-Depends: automake,
 # Make-Check   libxmlsec1-dev,
 # Crimson      libyaml-cpp-dev,
                lsb-release,
+# Make-Check   ninja-build,
```

Comment:
@tchaikov since in https://github.com/ceph/ceph/pull/39826/files#diff-221d0b3ec2fc135364e7faae118bd5ef661d07e2f373bba7456bf84318a27beaR17 we are using Ninja not just for make-check, should we drop # Make-Check mangling?

Reply:
i don't follow you. could you please enlighten me what we are using for in addition to "make check"?

Reply:
sure, here: https://github.com/ceph/ceph/pull/39826/files#diff-221d0b3ec2fc135364e7faae118bd5ef661d07e2f373bba7456bf84318a27beaR17 if I understand correctly, Ninja would be default build system for `do_cmake.sh` script, if we have `# Make-Check   ninja-build,` that should install ninja-build when are doing make check
```suggestion
#              ninja-build,
```

Reply:
the idea is to pull the dependencies as part of our CI. and "ninja-build" is not a must-have for building ceph packages, i don't want to drop "Make-Check".